src/Form/LegacyConsumer/Commands/SetupFieldPersistence.php

```diff
@@ -85,6 +86,10 @@ public function process( Field $field ) {
 						implode( '| ', array_values( array_filter( $data ) ) ) :
 						$data;
 
+					if( $field instanceof WpEditor ) {
+						$value = wp_kses_post( $data );
+					}
```

Comment:
Hmm. Sanitization and validation should be a part of the field definition. Since we don't have that right now, this is fine, but keep that in mind in the future.

Reply:
noted